initial updates for speakers and schedule

dzoladz opened the pull request:
This PR:
- adds speaker data
- adds talk posts
- adds schedule
- add several clock faces

Needs:
- [x] keynote sessions
- [x] speaker arrays in _posts
- [x] css improvements on schedule
- [ ] placeholder function

phette23 commented on assets/_scss/_presentations-workshops.scss:
```diff
@@ -8,14 +8,28 @@
 }
 
 .talk-row:nth-child(odd) {
```
I like this change and I think we can just delete the rule rather than comment it out.

dzoladz replied:
Removed rule

phette23 commented on assets/_scss/_presentations-workshops.scss:
```diff
@@ -8,14 +8,28 @@
+    background: $gray-300;
+}
+
+.talk-row h3 {
```
Would it make more sense to write `.talk-row .h4` so it's abstracted away from what element we use? Helps if we ever decide to change the header hierarchy. Actually, thinking about it more, we should probably add a new "talk-title" class in the HTML or something in addition to the .h4

dzoladz replied:
Resolved in commit b5dff8f0a69dab94a2d37107837a477fbde864ef

phette23 commented on assets/_scss/_timeline.scss:
```diff
@@ -196,7 +196,7 @@
 }
 
 .cd-timeline-content h2 {
```
Same comment as `.talk-row h3` in the other file: I think we should add a new, semantic class and target that rather than tie our rules to particular tags.

dzoladz replied:
Resolved in commit b5dff8f0a69dab94a2d37107837a477fbde864ef

phette23 commented:
@dzoladz OK I am going to read through every, or at least a bunch, of the proposals before finishing the review, but other than the talks content I have only two thoughts:

- [ ] the small CSS selector changes mentioned above
- [ ] the keynote pages e.g. http://127.0.0.1:4000/keynotes/lauren-chambers only have placeholder info, we definitely want something else in place before publishing. I will push a commit in a moment that just adds the bios we have and gives them generic titles for now.

Let me know if you want me to make the CSS changes I describe. I have a 2 pm meeting so I might not get to them for a while.

phette23 reviewed (changes requested):
OK I read a bunch of talks and looked at them rendered on the site and everything looked good! So it's just the CSS changes and I think we're ready.

dzoladz commented:
@phette23 Thanks. I'll review your CSS comments in more details this evening... and follow-up accordingly.

dzoladz commented:
@phette23 I've updated the css. I also noticed that speaker group 4 was displaying at the end of the day due because the leading `0` was missing from their time slot, so I've fixed that as well `time: 09:15-10:00`

phette23 commented:
@dzoladz Oh I realized too late you had "placeholder function" under the tasks for this PR. Is that something that cna be done separately or should I revert?
